### ISSUE TITLE
Contributing manual translation

### ISSUE THREAD

**Issue by agrexgh** [mesonbuild/meson#6893]
I'd like to contribute manual translation to Japanese,
but it seems that [Contribute guideline](https://mesonbuild.com/Contributing.html) didn't show how to do it.
Is there any plans to define guideline to translate manual?
Or can any contributers define new guideline?

**Comment by jpakkane** [comment]
Thank you for your interest in contributing. We don't have a policy yet for manual translations because this is the first time anyone has wanted to write one. We'll probably need to make up the rules as we go. Do you have a view on how the translation should live side by side on the web site? Does hotdoc have functionality for this? If not should we ask the developers for new features?

**Comment by agrexgh** [comment]
> Does hotdoc have functionality for this? If not should we ask the developers for new features?

Excuse me, I didn't know "hotdoc".
Did you mean below?
https://hotdoc.github.io/

And I'm sorry, because I'm not familiar with hotdoc, I can't tell you we need new feature now.
I must start studying hotdoc to judge that.

**Comment by jpakkane** [comment]
Yes, hotdoc is the way the web site is generated. The source files are in the git repository under the ´docs` subdirectory.

**Comment by agrexgh** [comment]
Thank you for replying.
I understood `hotdoc` is a kind of web site generator from source files(like *.md).

I think we don't need new feature of `hotdoc`
and we fork new repository for translating from this repo.
Many project seems that these have fork some project for translating each language.
At Vue.js project, it has each repository for each language(like jp.vue.js, cn.vue.js).

But I'm sorry that I'm a noob of github and manual translation.
I'll try forking new repository for japanese translation,
but I can't do everything.
I'd like someone to do following things.
1. Making translation rules.
1. Managing translation project(for example, approve pull request).
1. Publish manual website.

Does my comment answer your question?
